lib/saluki-components/src/sources/dogstatsd/mod.rs

```diff
@@ -837,7 +850,7 @@ async fn drive_stream(
 
 fn handle_frame(
     frame: &[u8], codec: &DogstatsdCodec, context_resolvers: &mut ContextResolvers, source_metrics: &Metrics,
-    peer_addr: &ConnectionAddress, enabled_filter: EnablePayloadsFilter,
+    peer_addr: &ConnectionAddress, enabled_filter: EnablePayloadsFilter, additional_tags: &Arc<[String]>,
```

Comment:
I'd make this one be `&[String]` as well, since we don't need to know, at this point, that it's actually an `Arc`.